android/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/Biometrics095Service.java

```diff
@@ -109,6 +134,15 @@ public List<BiometricsDto> handleRCaptureResponse(Modality modality, InputStream
                         false,
                         1, 0,
                         captureDto.getQualityScore()));
+
+                if(RegistrationConstants.ENABLE.equalsIgnoreCase(this.globalParamRepository
+                        .getCachedStringGlobalParam(RegistrationConstants.DEDUPLICATION_ENABLE_FLAG))) {
+                    boolean isMatched = MatchUtil.validateBiometricData(modality, captureDto, biometricsDtoList, userBiometricRepository, iBioApiV2);
+                    if(isMatched){
+                        Log.i(TAG, "Biometrics Matched With Operator Biometrics, Please Try Again");
+                        return null;
```

Comment:
we should return empty list instead of null and check if its empty in parse response, functionality should remain the same nonetheless

Reply:
Added empty list in place of null and changed respective conditions